p2panda-js/README.md

```diff
@@ -74,16 +75,39 @@ const session = new Session('https://welle.liebechaos.org').setKeyPair(keyPair);
 
 // Compose your operation payload, according to chosen schema
 const payload = {
-  operation: 'Hi there',
+  message: 'Hi there',
 };
 
 // Send new chat operation to the node
 await session.create(payload, { schema: CHAT_SCHEMA });
 
 // Query instances from the p2panda node
-const instances = await session.query({ schema: CHAT_SCHEMA });
-console.log(instances[0].operation);
-// -> 'Hi there'
+import { gql, useQuery } from '@apollo/client';
+
+const GET_CHAT_MESSAGES = gql`
+  query GetChatMessages {
```

Comment:
I think there is a shorthand form where you can even skip the query bracket, looks even simpler then!